### ISSUE TITLE
Cannot import name 'soft_unicode' from 'markupsafe'

### ISSUE THREAD

**Issue by LuanComputacao** [scanapi/scanapi#528]
### Environment
- Operating System: LInux
- Python version: 3.10.2

### Description of the bug 
```shell
❯ scanapi --version
Traceback (most recent call last):
  File "/home/luancomputacao/api_tests/venv/bin/scanapi", line 5, in <module>
    from scanapi import main
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/scanapi/__init__.py", line 1, in <module>
    from scanapi.__main__ import main
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/scanapi/__main__.py", line 8, in <module>
    from scanapi.scan import scan
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/scanapi/scan.py", line 13, in <module>
    from scanapi.reporter import Reporter
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/scanapi/reporter.py", line 10, in <module>
    from scanapi.template_render import render
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/scanapi/template_render.py", line 2, in <module>
    from jinja2 import Environment, FileSystemLoader, PackageLoader
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/home/luancomputacao/api_tests/venv/lib/python3.10/site-packages/markupsafe/__init__.py)
```

### Expected behaviour?
Print the scanapi version

### How to reproduce the bug?
* install Python 3.10.2
* install scanapi
* run `$ scanapi --version`

**Comment by leonardomaier** [comment]
Looks like the issue is due to upgrade to `MarkupSafe:2.1.0` where they have removed soft_unicode.

Release note: https://markupsafe.palletsprojects.com/en/2.1.x/changes/#version-2-1-0

**Comment by Ronald-TR** [comment]
> Looks like the issue is due to upgrade to `MarkupSafe:2.1.0` where they have removed soft_unicode.
> 
> Release note: https://markupsafe.palletsprojects.com/en/2.1.x/changes/#version-2-1-0

You are right. I fix that downgrading MarkupSafe to version 2.0.1.

```
pip uninstall MarkupSafe
pip install MarkupSafe==2.0.1
```
